cmd/aperturectl/cmd/root.go

```diff
@@ -92,7 +90,28 @@ func Execute() {
 		os.Exit(1)
 	}
 
-	if err := RootCmd.Execute(); err != nil {
+	newer, err := utils.IsCurrentVersionNewer(info.Version)
+	if err != nil {
+		log.Error().Err(err).Msg("Failed to check if current version is newer")
+	}
+	if newer {
+		err = blueprints.RemoveRunE(nil, nil)
+		if err != nil {
+			log.Error().Err(err).Msg("Failed to remove latest blueprints")
+		}
+		err = blueprints.PullRunE(nil, nil)
+		if err != nil {
+			log.Error().Err(err).Msg("Failed to pull latest blueprints")
+		} else {
+			err = utils.UpdateVersionFile(info.Version)
+			if err != nil {
+				log.Error().Err(err).Msg("Failed to update version file")
+			}
+		}
+		_ = blueprints.PullPostRunE(nil, nil)
+	}
```

Comment:
I guess we just need to remove `.aperturectl/{blueprints,aperture}` directories. The pull should happen automatically in the sub-commands.

Reply:
Addressed in [20f71ca](https://github.com/fluxninja/aperture/pull/2748/commits/20f71ca74f92d71eb37081189b360ee8ba1613f4)